platform/src/main/java/org/stellar/anchor/platform/utils/RequestLoggerFilter.java

```diff
@@ -0,0 +1,98 @@
+package org.stellar.anchor.platform.utils;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.Objects;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.web.filter.OncePerRequestFilter;
+import org.springframework.web.util.ContentCachingRequestWrapper;
+import org.springframework.web.util.ContentCachingResponseWrapper;
+import org.stellar.anchor.util.Log;
+
+/**
+ * Log each request and response. The request body is never logged and the response body is only
+ * logged when it's an error
+ *
+ * @see <a href="https://stackoverflow.com/a/42023374/875657">StackOverflow Answer</a>
+ */
+public class RequestLoggerFilter extends OncePerRequestFilter {
+  @Override
+  protected void doFilterInternal(
+      HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
+      throws ServletException, IOException {
+
+    long startTime = System.currentTimeMillis();
+    StringBuffer logMessage =
+        new StringBuffer("\n\tREST Request:\n")
+            .append("\t\t[METHOD: " + request.getMethod() + "]\n")
```

Comment:
i would avoid newline characters, i believe each line registers as a separate log message and needs to be merged on the backend for systems like elasticsearch. not sure what the overhead is for this

Reply:
That's a great point! I'll update the format to make it a one-liner.

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/519/commits/1bf7900297fe5e3c8d0d1c1f793a58d7efacce65.